qa/tasks/qemu.py

```diff
@@ -496,17 +499,20 @@ def run_qemu(ctx, config):
                 continue
 
             if disk['encryption_format'] == 'none':
+                interface = 'virtio'
                 disk_spec = 'rbd:rbd/{img}:id={id}'.format(
                     img=disk['image_name'],
                     id=client[len('client.'):]
                     )
             else:
+                interface = 'ide'
```

Comment:
@orozery can you keep a comment somewhere or add keep a open tracker just to remind us to remove this when librbd with encryption is available?

Reply:
opened https://tracker.ceph.com/issues/53592